asmcomp/cmm_helpers.mli

```diff
@@ -188,7 +188,7 @@ val return_unit : Debuginfo.t -> expression -> expression
 val remove_unit : expression -> expression
 
 (** Blocks *)
-val mk_load_mut : memory_chunk -> operation
+val mk_load_mut : is_atomic:bool -> memory_chunk -> operation
```

Comment:
Given that the vast majority of loads are non-atomic, it would make sense to make this an optional argument with `false` as the default argument.

Reply:
Or just leave `mk_load_mut` as it is today and introduce `mk_load_atomic`.

Reply:
> Or just leave `mk_load_mut` as it is today and introduce `mk_load_atomic`.

Done in the new version.